docs/fastddsgen/pubsub_app/includes/first_app.rst

```diff
@@ -42,13 +32,16 @@ The `-example` option creates an example application, and the CMake files needed
 
             <path-to-Fast-DDS-workspace>/src/fastrtpsgen/scripts/fastrtpsgen.bat -example CMake HelloWorld.idl
 
-        - However, if you have followed the **manual installation**, run:
+        - However, if you have followed the **stand-alone installation**, run:
 
         .. code:: bash
 
             <path-to-Fast-DDS-Gen>/scripts/fastrtpsgen.bat -example CMake HelloWorld.idl
```

Comment:
Where is this if binary installation? Maybe a note stating it?

Reply:
Check Windows installation directories.

Reply:
Maybe we can point to somewhere where it say where `path-to-Fast-DDS-Gen` is if installing from binaries